dashboards/state-sync-v2.json

```diff
@@ -3511,8 +4078,8 @@
       {
         "current": {
           "selected": false,
-          "text": "aptos-devnet",
-          "value": "aptos-devnet"
+          "text": "sherryxnet",
```

Comment:
Is this an intended change?

Reply:
Annoyingly not. 😭 I made these changes on the `sherryxnet` deployment because devnet monitoring is down and I guess this is what we get when we have to copy/pasta the json. I'll wait for the devnet monitoring pod to come up to re-sync them.